fsw/inc/cs_tbldefs.h

```diff
@@ -81,7 +81,8 @@ typedef struct
 {
     cpuaddr StartAddress;       /**< \brief The Start address to Checksum */
     uint16  State;              /**< \brief Uses the CS_STATE_... defines from above */
-    uint16  ComputedYet;        /**< \brief Have we computed an Integrity value yet */
+    bool    ComputedYet;        /**< \brief Have we computed an Integrity value yet */
+    bool    Filler8;            /** <\brief Padding */
```

Comment:
Same comment here, this should stay as `uint16` to be explicit.  This file is an interface.